src/main/java/me/clip/placeholderapi/expansion/manager/LocalExpansionManager.java

```diff
@@ -323,6 +327,19 @@ public boolean unregister(@NotNull final PlaceholderExpansion expansion) {
     if (expansions.remove(expansion.getIdentifier().toLowerCase(Locale.ROOT)) == null) {
       return false;
     }
+    
+    // Don't unregister expansions that are marked internal or set to be persistent.
+    if (expansion.getExpansionType() == Type.INTERNAL || expansion.persist()) {
```

Comment:
Shouldn't this check be made before the removal from the map on line 327?

Reply:
true

Reply:
Moved it